monte_carlo.ipynb

```diff
@@ -39,7 +39,14 @@
     "numberOfCustomerEvents = round(1.1*timeHorizon/meanTimeBetweenCustomerArrival)\n",
     "prices = np.arange(0,15.1,0.1)\n",
     "watchedCompany = 0\n",
-    "numberSimulations = 1000"
+    "numberSimulations = 100  # value used for paper: 10000\n",
```

Comment:
Now we are no longer in sync with the market_situations notebook, correct? That notebook still runs for hours.

Reply:
The market_situations notebook runs 15 minutes currently. I'm not sure what you mean with in sync?